auth/src/main/java/com/firebase/ui/auth/ui/credentials/CredentialSaveActivity.kt

```diff
@@ -0,0 +1,89 @@
+package com.firebase.ui.auth.ui.credentials
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import com.firebase.ui.auth.IdpResponse
+import com.firebase.ui.auth.data.model.FlowParameters
+import com.firebase.ui.auth.data.model.Resource
+import com.firebase.ui.auth.ui.InvisibleActivityBase
+import com.firebase.ui.auth.util.ExtraConstants
+import com.firebase.ui.auth.viewmodel.ResourceObserver
+import com.firebase.ui.auth.viewmodel.credentialmanager.CredentialManagerHandler
+import com.google.android.gms.auth.api.credentials.Credential
+import androidx.lifecycle.ViewModelProvider
+import com.google.firebase.auth.FirebaseAuth
+
+class CredentialSaveActivity : InvisibleActivityBase() {
+
+    companion object {
```

Comment:
Can we leave companions objects at the end of the class, conforming to [Kotlin's coding conventions](https://kotlinlang.org/docs/coding-conventions.html#class-layout)?